browser/ui/brave_browser_command_controller.cc

```diff
@@ -313,6 +315,11 @@ bool BraveBrowserCommandController::ExecuteBraveCommandWithDisposition(
     case IDC_MANAGE_BRAVE_VPN_PLAN:
       brave::OpenBraveVPNUrls(browser_, id);
       break;
+#if BUILDFLAG(ENABLE_SIDEBAR)
```

Comment:
I don't think this buildflag is needed since sidebar is tied to toolkit views and wouldn't make any sense outside of it. I think we should drop it because the usage is confusing. If you need to guard sidebar outside of ui/views use TOOLKIT_VIEWS